src/lib/containers/SubsectionRowRenderer.tsx

```diff
@@ -1,7 +1,6 @@
 import React, { useState } from 'react'
 import {
-  insertConditionsFromSearchParams,
-  KeyValue,
```

Comment:
Tried to remove this `KeyValue` type because it's the same as builtin `Record<string, string>`